tests/src/JIT/Directed/Misc/gettype/gettypetypeofmatrix.csproj

```diff
@@ -22,8 +22,7 @@
     </CodeAnalysisDependentAssemblyPaths>
   </ItemGroup>
   <PropertyGroup>
-    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
-    <DebugType></DebugType>
+    <DebugType>None</DebugType>
```

Comment:
Sure. It is equal for windows-based msbuild. On Linux we are getting error instead:

> 3754>CSC : error CS0041: Unexpected error writing debug information -- 'Windows PDB writer is not available -- could not find Microsoft.DiaSymReader.Native.amd64.dll' [/home/coreclr/git/coreclr/tests/src/JIT/Directed/Misc/gettype/gettypetypeofmatrix.csproj]